README.md

```diff
@@ -16,9 +16,125 @@
 ```
 
 ## Install
-```sh
+
+```bash
 npm install @railgun-reloaded/circuits-v2
 ```
 
+## Usage
```

Comment:
Ok! However, perhaps you can reread this again now, and consider if it's suitable. The reason I put it that way is, what I personally would be looking for is; what are the inputs and outputs I get. Those are of course specified by the types, but if I go read the function definition for `prove` and `verify`, it does not make me much wiser. I do not mean that we need to explain what the `merkleRoot` is or how `token` is specifically derived, but enough information that someone with context of the protocol knows where to get this information and what to plug in where